### DISCUSSION TITLE
which human genome reference does mixcr use?

### DISCUSSION THREAD

**Discussion by xiaoyu462** [milaboratory/mixcr#1610] [Q&A] [answered]
Hi, I have the alignments.vdjca file with me, and I wish to know the human genome reference used by mixcr. I want to use IGV to view the read alignments to the reference. Thanks!

**Comment by mizraelson** [comment, marked as answer]
Hi, MiXCR uses its own reference which is based on the genome assembly. The version of the genome used to create the reference depends on the MiXCR version you are using. You can use our interactive library database to export the IGV reference sequences:
https://vdj.online/library

**Reply by xiaoyu462** [reply]
Thank you very much for the reply! 

Then may I know how to convert the vdjca file into bam file for viewing on IGV? 

And if there is a way to visualize the text file which is from running "exportAlignmentsPretty" on mixcr in IGV or other platforms? 

My exported text or tsv files do not contain chr information. Will it be fine?

Thanks again!

**Reply by mizraelson** [reply]
You can't convert `vdjca` to bam directly as they are different formats. `vdjca` is a MiXCR-specific format designed for TCR/BCR genes alignments, while bam is a binary format for storing sequence data. However, you can use `bam` files as input for `mixcr align`. To extract data from a `vdjca` file, you have two options: `exportAlignments` and `exportAlignmentsPretty`. The fields available for `exportAlignments` can be found in the [MiXCR documentation](https://mixcr.com/mixcr/reference/mixcr-export/#export-fields). MiXCR focuses on immune receptor gene analysis, which doesn't require chromosome information; therefore, chromosome (chr) information is not typically included in `vdjca` exports.
